README.md

```diff
@@ -13,9 +12,11 @@
     <img src="https://img.shields.io/badge/license-Apache2-blue.svg?style=flat" alt="Apache 2">
 </p>
 
-
 # Create and deploy a Go Web Application using Gin
 
+[![Go Report Card](https://goreportcard.com/badge/github.com/alessiosavi/go-web-app)](https://goreportcard.com/report/github.com/alessiosavi/go-web-app) [![Codacy Badge](https://api.codacy.com/project/badge/Grade/0e3ca10e3066494f9517cd93a7a5d1a5)](https://www.codacy.com/manual/alessiosavi/go-web-app?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=alessiosavi/go-web-app&amp;utm_campaign=Badge_Grade) [![GoDoc](https://godoc.org/github.com/IBM/go-web-app?status.svg)](https://godoc.org/github.com/IBM/go-web-app) [![License](https://img.shields.io/github/license/IBM/go-web-app)](https://img.shields.io/github/license/IBM/go-web-app) [![Code size](https://img.shields.io/github/languages/code-size/IBM/go-web-app)](https://img.shields.io/github/languages/code-size/IBM/go-web-app) [![Repo size](https://img.shields.io/github/repo-size/IBM/go-web-app)](https://img.shields.io/github/repo-size/IBM/go-web-app) [![Issue open](https://img.shields.io/github/issues/IBM/go-web-app)](https://img.shields.io/github/issues/IBM/go-web-app)
```

Comment:
We also cant link anything from your personal github

Reply:
Of course, this can be changed to the real repository. In order to have a preview, i had to link to my repository fork.